scripts/lib/releaseNotes.test.ts

```diff
@@ -0,0 +1,94 @@
+// This code is a Qiskit project.
+//
+// (C) Copyright IBM 2023.
+//
+// This code is licensed under the Apache License, Version 2.0. You may
+// obtain a copy of this license in the LICENSE file in the root directory
+// of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// Any modifications or derivative works of this code must retain this
+// copyright notice, and modified files need to carry a notice indicating
+// that they have been altered from the originals.
+
+import { describe, expect, test } from "@jest/globals";
+import {
+  sortReleaseNotesVersions,
+  extractMarkdownReleaseNotesPatches,
+} from "./releaseNotes";
+
+describe("sortReleaseNotesVersions", () => {
+  test("Test versions have the correct order", () => {
```

Comment:
No need to use `describe` since it's a single test. You can use `test` at the top-level.